main.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
any reason why the script is in bash?

Reply:
yes, the command lines (both for file uploading and job submission) recommended by Archer API documentation are in bash. Also the sample list preparation for the job submission is quite convenient in bash.